playgrounds/vite/src/main.tsx

```diff
@@ -1,8 +1,12 @@
 import * as React from 'react';
 import { createRoot } from 'react-dom/client';
-import styled from '@emotion/styled';
+
+// Workaround for emotion/styled not working in ESM: https://github.com/emotion-js/emotion/issues/2730#issuecomment-1205489497
+import _styled from '@emotion/styled';
+const styled = _styled.default;
```

Comment:
ahhh i don't like this. it's ridiculous that this library does not support ESM in 2023; we should get rid of it (in a separate PR probably)

Reply:
After https://github.com/iTwin/iTwinUI/pull/1549/commits/efb929b1139205be700a7ade69bd63668ab9afdd (https://github.com/iTwin/iTwinUI/pull/1549#discussion_r1318825306), this change was no longer needed, and so was reverted.

Yes, maybe we can stop using `emotion/styled` in a future PR since it is only used in this file.